languages/elixir/exercises/concept/booleans/.meta/design.md

```diff
@@ -0,0 +1,46 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of booleans and logicical expressions in Elixir through the expression of game rule logic(1).
+
+## Learning objectives
+
+- Know what a variable is.
+- Know how to define a named function.
+- Know how to define a function with parameter variables.
+- Know how to return a value from a function.
+- Know how to define a boolean.
+- Know how to use logical operators on booleans.
+  - `and/2`, `or/2`, `not/1`
+- Know how to name functions that return boolean values.
+
+## Out of scope
+
+- Default parameters.
+- Truthy logical comparisons using `&&/2`, `||/2`, `!/1`
+- Single-line functions
+- Booleans as special atoms
+
```

Comment:
You are missing the `## Concepts` section (see [example](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/booleans/.meta/design.md#concepts)).

Reply:
Oops! Will add